### ISSUE TITLE
Bad field types on response JSON object (errors 607, 614, 701)

### ISSUE THREAD

**Issue by crd1985** [wazuh/wazuh-api#277]
Hello!

I have detected a problem when I send this request to 3.8 Wazuh API version
```bash
# curl -u foo:bar -X POST -H 'Content-type: application/xml' -d @agent.conf.xml "http://localhost:55000/agents/groups/webserver/files/agent.conf?pretty"
{"error":"701","message":"Size of XML file is too long"}
```
The "701" (string) should be 701 (integer) and the JSON should be prettified.

The problem seems to be in app.js.

Regards,
Dani

**Comment by mgmacias95** [comment]
This issue will be deprecated after Wazuh 4.0 when we do the migration to OpenAPI.

**Comment by druizz90** [comment]
Hi team,

I suggest to cast error codes to integers instead strings in exceptions that we raised in `app.js` with `res_h.bad_request` function (as we do in API controllers) for solving this issue.

I made this fix and ran `mocha` tests and all is OK.

**Comment by druizz90** [comment]
Hi team,

I suggest to cast code errors thrown in `app.js` to integers instead strings (as we do with the other exceptions we raise with through res_h.bad_request.

I made this change and `API` works fine and `mocha` tests run properly.

**Comment by druizz90** [comment]
This issue was solved in #400.
